discord_bot/discord_bot.py

```diff
@@ -20,13 +21,20 @@ def __init__(self):
         self.gateway_ws_url = None
         self.heartbeat_interval_ms = None
         self.last_seq = None
+        self.logger = None
         self.event_loop = None
         self.websocket = None
         self.session_id = None
 
-    # TODO: Docstrings for setup and run
+    # TODO: Docstrings for setup and runs
     def setup(self):
         self.config = self.load_config(CONFIG_FILE_PATH)
+
+        logging.basicConfig(format='%(asctime)s %(module)s %(levelname)s: %(message)s',
+                            datefmt='%m/%d/%Y %I:%M:%S %p',
+                            level=logging.INFO)
+        self.logger = logging.getLogger(__name__)
```

Comment:
We'll want to set `self.logger`'s level instead of configuring it in the root logger, so let's do `self.logger.setLevel(self.config["log_level"])` like before and add the changes back to the `config.json` files.